### PR TITLE
Revert "Bump mysql-connector-java from 8.0.12 to 8.0.16 in /restServices"

### PR DESCRIPTION
**Pull request by agrawal-utkarsh**
Reverts kh-satyam/StudentInformationSystem#1

### CODE REVIEW

**Comment by agrawal-utkarsh** [comment]
@kh-satyam  please review this PR on priority.

File: restServices/pom.xml [hunk 1]
```diff
@@ -59,7 +59,7 @@
 		<dependency>
     		<groupId>mysql</groupId>
     		<artifactId>mysql-connector-java</artifactId>
-    		<version>8.0.16</version>
+    		<version>8.0.12</version>
```
Comment (kh-satyam) [inline restServices/pom.xml:62]: Why are we downgrading to the lower version of mysql-connector?

Comment (kh-satyam) [reply restServices/pom.xml:62]: is this code tested ?

**Review by kh-satyam — CHANGES_REQUESTED** [review]
reviewed

File: restServices/pom.xml (hunk 1 above)
Comment (jagmtsandhu) [reply restServices/pom.xml:62]: you have downgraded the version!!

**Comment by kh-satyam** [comment]
@jagmeet787  Please avoid commenting on comments of the reviewer and adding all the possible emojis and you missed 5 of them

**Comment by kh-satyam** [comment]
@agrawal-utkarsh  Please correct the changes requested , since this is a security bug , and need to be closed.

**Comment by kh-satyam** [comment]
@ Abhay2608

**Review by kh-satyam — CHANGES_REQUESTED** [review]
Please correct the PR title

**Comment by agrawal-utkarsh** [comment]
> @agrawal-utkarsh Please correct the changes requested , since this is a security bug , and need to be closed.

Yes, you are right, we dont need to downgrade the version. I am closing the pr. As for testing the code, i think we need to have a meeting to discuss the relevant test cases with input and output. Also what sort of testing is needed. Please prepare a detailed doc and set up a meeting in Jio Meet

**Comment by kh-satyam** [comment]
@agrawal-utkarsh  I believe no detailed documentation is required, just test the db-connectivity and end-to-end flow, with the bump version of mysql.

**Pull request closed without merging**